frontend/app/framework/angular/forms/editors/code-editor.component.ts

```diff
@@ -82,8 +82,13 @@ export class CodeEditorComponent extends StatefulControlComponent<undefined, str
         if (this.height) {
             this.editor.nativeElement.style.height = `${this.height}px`;
         }
-
-        this.resourceLoader.loadScript('https://cdnjs.cloudflare.com/ajax/libs/ace/1.2.6/ace.js').then(() => {
+        let url = 'dependencies/ace/ace.min.js';
+        if (process.env.NODE_ENV !== 'production') {
+            url = 'https://localhost:3000/' + url;
+        } else {
+            url = 'build/' + url;
+        }
+        this.resourceLoader.loadScript(url).then(() => {
```

Comment:
Can we made this as apart of the resource loader? 

E.g. loadLocalScript and loadLocalStyle

And also use string interpolation please, I just always use it ;)